src/modules/core/components/SingleUserPicker/SingleUserPicker.tsx

```diff
@@ -224,6 +229,11 @@ const SingleUserPicker = ({
                   {value.profile.displayName ||
                     value.profile.username ||
                     value.profile.walletAddress}
+                  {showMaskedAddress && (
+                    <span className={styles.maskedAddress}>
+                      <MaskedAddress address={value.profile.walletAddress} />
+                    </span>
```

Comment:
I'm not really happy with showing a masked address in the single user picker.

But this is not a problem with the code you wrote, this is more of a problem with the actual specs.